contracts/query-queue/tests/integration.rs

```diff
@@ -40,7 +40,7 @@ static QUEUE_WASM: &[u8] =
     include_bytes!("../../queue/target/wasm32-unknown-unknown/release/queue.wasm");
 
 fn create_contract() -> (Instance<MockApi, MockStorage, MockQuerier>, MessageInfo) {
-    let gas_limit = 500_000_000; // enough for many executions within one instance
+    let gas_limit = 1_000_000_000_000; // ~1ms, enough for many executions within one instance
```

Comment:
Why does the gas increase very big?

Reply:
it follows https://github.com/CosmWasm/cosmwasm/pull/1120